components/sidebar/sidebar_service.cc

```diff
@@ -32,7 +32,7 @@ SidebarItem GetBuiltInItemForType(SidebarItem::BuiltInItemType type) {
           GURL("https://talk.brave.com/"),
```

Comment:
Can we make this `https://talk.brave.com/widget`? This way, when you're logged in, clicking the video icon will actually create a new video conference (if no open tabs are found) 😄

Reply:
Updated.

Reply:
Hmm, url is redirected from https://talk.brave.com/widget to https://talk.brave.com/ when icon clicks.
but typing https://talk.brave.com/widget doesn't redirect. strange..

Reply:
> Can we make this `https://talk.brave.com/widget`? This way, when you're logged in, clicking the video icon will actually create a new video conference (if no open tabs are found) 😄

Hmm, a little bit confused.  you mean clicking talk icon should create new talk tab if there are only logged in talk?

Reply:
Done!